crawler/config/defaults.py

```diff
@@ -38,7 +38,7 @@
 ###
 # Baracoda
 ###
-BARACODA_BASE_URL = f"http://{LOCALHOST}:8000"
+BARACODA_BASE_URL = f"http://{LOCALHOST}:7900"
```

Comment:
why this change?

Reply:
Oh yeah sorry.  It's partly because I have to make this change manually every time I hook it all up locally.  If you look just below the fold here you'll see that we're using the same port for Cherrytrack which is clearly never going to work.